scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -0,0 +1,1335 @@
+# pylint: disable=anomalous-backslash-in-string, global-variable-undefined, global-variable-not-assigned
+# pylint: disable=invalid-name, too-many-arguments, superfluous-parens, too-many-branches, redefined-builtin
```

Comment:
Oh bad surprise that we have `too-many-branches` in this file. I thought that we wouldn't have this kind of issues in these routines which looked relatively well organized and modular.
This is probably because of `focus_v2`?